packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -434,16 +636,54 @@ const Combobox = forwardRef<HTMLDivElement, ComboboxProps>(
         </label>
         {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
         <TextFieldWrapper
-          className={classnames({ 'TextFieldWrapper--error': hasError })}
+          className={classnames({
+            'Combobox__TextFieldWrapper--multiselect': multiselect,
+            'TextFieldWrapper--error': hasError
+          })}
           // We're handling click here to open the listbox when the wrapping element is clicked,
           // there's already keyboard handlers to open the listbox on the input element
           onClick={handleInputClick}
         >
+          {multiselect &&
+            selectedValues.map((value, idx) => {
```

Comment:
Nit: Same suggestion as above regarding index...

```suggestion
            selectedValues.map((value, index) => {
```

Reply:
Updated